src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "microsoft.aspnetcore.razor.vscode",
   "private": true,
-  "version": "0.0.1",
+  "version": "6.0.0-dev",
```

Comment:
?

Reply:
Happens automatically sometimes, don't think it was intentionally committed.

Reply:
Yeah, if we could find a way to stop it from making this edit every time you do `.\build.cmd` that would be really nice.